setup.py

```diff
@@ -0,0 +1,62 @@
+"""setuptools based setup module.
+
+Adapted from https://github.com/pypa/sampleproject/blob/main/setup.py
+"""
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+import pathlib
+import versioneer
+
+here = pathlib.Path(__file__).parent.resolve()
+
+# Get the long description from the README file
+long_description = (here / "README.md").read_text(encoding="utf-8")
+
+setup(
+    name="timemachine",
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    description="A high-performance differentiable molecular dynamics, docking and optimization engine",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/proteneer/timemachine",
+    author="Relay Therapeutics",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3",
+    ],
+    keywords="molecular dynamics",
+    packages=find_packages(),
+    python_requires=">=3.7",
+    install_requires=[
+        "bootstrapped",
+        "grpcio",
+        "grpcio-tools",
+        "hilbertcurve",
+        "jax",
+        "jaxlib",
+        "networkx",
+        "numpy",
+        "pymbar",
+        "pyyaml",
+        "scipy",
+    ],
+    extras_require={  # Optional
+        "dev": ["black==21.10b0", "pre-commit"],
+        "test": ["pytest", "pytest-cov"],
+    },
+    # package_data={
```

Comment:
This could be a good way to include data files (non-python files, or python files that aren't part of a module)

Reply:
`datasets/`, `testsystems/`, and forcefield files come to mind -- maybe others?

Reply:
I don't think we want to package anything up beyond what is in `timemachine/`? Though really depends on what we want from the python package. Is it a testable python package on its own, or do we want to require the git repo along with the python package.

Reply:
I'm leaning toward leaving these out of the package for now, since it seems like we might want to have a separate discussion about what's appropriate to include; I'll leave the commented-out lines in case we do want to add package data in the future.